.internal-ci/helm/fog-services/templates/fog-view-grpc-ingress.yaml

```diff
@@ -25,6 +25,6 @@ spec:
         pathType: Prefix
         backend:
           service:
-            name: {{ include "fogServices.fullname" . }}-fog-view
+            name: {{ include "fogServices.fullname" . }}-fog-view-router
```

Comment:
FWIW, I think this will break the ingress connection to the existing "loaded" view instances when applied.

Reply:
Yes, this is where we will have to deploy the stand-alone PREVIOUS ledger and view to match the selector within that service name.  That will allow access during the upgrade, with no downtime.